src/app/marketplace/marketplace-product-details.component.html

```diff
@@ -33,52 +45,101 @@ <h1>{{ product.title }}</h1>
         [availableActions]="availableActions"></app-marketplace-action-buttons>
     </section>
 
+    <section class="product-details__summary">
+      <a
+        *ngIf="sellerRating.toNumber() > 0"
+        (click)="scrollIntoView('seller-rating')"
+         class="product-details__summary-item product-details__summary-item_link">
+        <span class="product-details__summary-number">
+          {{ sellerRating.toNumber() | percent }}
+        </span>
+        <span class="product-details__summary-label">
+         Positive feedback
+        </span>
+      </a>
+
+      <div class="product-details__summary-item" *ngIf="sellerRating.toNumber() == 0">
+        <span class="product-details__summary-label">
+         No feedback yet
+        </span>
+      </div>
+
+      <a *ngIf="product.sampleFile?.length > 0"
+        (click)="scrollIntoView('sample-files')"
+        class="product-details__summary-item product-details__summary-item_link">
+        <span class="product-details__summary-number">
+          {{ product.sampleFile.length }}
+        </span>
+        <span class="product-details__summary-label" [ngPlural]="product.sampleFile.length">
+         Sample
+          <ng-template ngPluralCase="=0">files</ng-template>
```

Comment:
This case is not needed because you displaying 'No sample files provided' if sampleFile.length is equal to 0.